src/redux/types/messages.js

```diff
@@ -0,0 +1,2 @@
+export const SET_MESSAGE = 'SET-MESSAGE';
+export const ADD_MESSAGE = 'ADD-MESSAGE';
```

Comment:
Please use "_" underscore for the constants instead of "-" =)

Reply:
It will be consistent with the names =)

Reply:
Done!